README.md

```diff
@@ -350,7 +343,19 @@ Getting the authentication token by calling `requestAuthToken()` with the OAuth
   const clientId = /* OAuth Client's id, best to put it in .env file and use it from there */;
   const clientSecret = /* OAuth client's secret, best to put it in .env file and use it from there */;
   const authToken = await requestAuthToken(clientId, clientSecret);
+
+  const before = isAuthTokenSet(); // false
   setAuthToken(authToken);
+  const after = isAuthTokenSet(); // true
+```
+
+Alternatively, authentication token can be set on a per-request basis, which also overrides any global (`setAuthToken()`) token:
```

Comment:
I would write that differently - **... overrides any global token that was set by / with `setAuthToken()`**

Reply:
Better, thank you! Fixed.

Reply:
Nice. Resolving